main.go

```diff
@@ -219,3 +224,16 @@ func (m model) moveDown() tea.Cmd {
 
 	return nil
 }
+
+func playMusic(){
+	f,err :=os.Open("assets/bgm.mp3")
```

Comment:
Let's format the file correctly. There should be a space between the comma and err.

```
f, err := os.Open(...)
```

Reply:
Does this cause any indentation errors like in python or is it just standard practice?